wso2is_analytics/README.md

```diff
@@ -16,13 +16,18 @@ environments. It supports multiple versions of WSO2 Data Analytics Server. Confi
 
 * Setup the puppet environment with the puppet modules wso2is_analytics and wso2base.
 * WSO2 IS Analytics 5.4.0  puppet modules are compatible and tested with
```

Comment:
Let's tell "WSO2 IS Analytics 5.4.0 puppet module is compatible and tested with"